pkg/monitoring/rules/alerts/prometheus.go

```diff
@@ -0,0 +1,51 @@
+package alerts
+
+import (
+	"errors"
+	"fmt"
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
```

Comment:
We usually split imports into 3 blocks:
- Standard library
- External imports specified in `go.mod` file.
- Imports of packages in this project.

Can you move this line into the block below?